bats/helpers/ledger.bash

```diff
@@ -2,7 +2,7 @@ CURRENT_FILE=${BASH_SOURCE:-bats/helpers/.}
 source "$(dirname "$CURRENT_FILE")/_common.bash"
 source "$(dirname "$CURRENT_FILE")/cli.bash"
 
-METRICS_ENDPOINT="localhost:3002/metrics"
+METRICS_ENDPOINT="localhost:3003/metrics"
```

Comment:
again this.. why modify exporter endpoint if the new addition in galoy pay?

Reply:
See above